### ISSUE TITLE
Showcase: <https://simplx-web.herokuapp.com>

### ISSUE THREAD

**Issue by angminsheng** [Stanko/react-plx#50]
Hi Stanko ! We are a group of three junior web developer who has just about to finish our 9 weeks coding bootcamp. We have been learning react for two weeks and we really liked and enjoyed your React Plx package as the code is clean and easy to work with. We decided to make a website that showcases all your parallax effects and people willl be able to export code snippet based on the effects they have chosen. If you have time we would love you to take a look and give us some feedback . Thank you once again for making this awesome package!

best regards,
Min

email : angmin92@gmail.com

**Comment by Stanko** [comment]
Hello @angminsheng,
You guys made my day :) this is brilliant. Make sure you add a description so people know what the app does. It took me a minute to figure it out myself.

Sometimes, app breaks (I didn't dig deep examining errors in the console), so you might want to use React's [Error Boundaries](https://reactjs.org/docs/error-boundaries.html).

Personally I would open source something like this, but it is up to you. People using it would be able to improve it through PRs.

Also I think it could benefit from small UX tweaks, but nothing major.

Bottom line is - I love it :) keep up with good work!

Cheers!

**Comment by angminsheng** [comment]
Hi Stanko, 

Glad to hear that you like it =)

We are still working on the project and I think there are some bugs that we are fixing at the moment. We have a presentation tomorrow where other teams will showcase their projects as well. It would mean a lot for us if you can feature it on the npm website so other people can try it out because this is why we started this project at the first place. We want to make animations and parallax effect more accessible to new developer like how your package simplifies our life =)

Thanks for taking your time to look at it and for the feedback as well.

Cheers!

**Comment by Stanko** [comment]
I just added it to the [readme](https://github.com/Stanko/react-plx#user-content-simplx) 🎉

**Comment by angminsheng** [comment]
Hi Stanko, thank you so much. We also added your github repository on our small blog section and tutorial section so that people will know where to get the documentation of your package.
